src/mmw/mmw/settings/layer_settings.py

```diff
@@ -197,6 +197,8 @@
                         'HUC-8 to analyze.',
             'minZoom': 7,
             'selectable': True,
+            'searchable': True,
+            'search_priority': 100,
```

Comment:
I think these should be sorted by relevance. Since we're not sure which boundaries are most relevant yet, I decided to set the sort order explicitly, which seems like the most flexible approach. This initial configuration prioritizes small/local boundaries over large ones.